### PR TITLE
Tabs justify

### PR DESCRIPTION
**Pull request by jwijay**

### CODE REVIEW

File: src/scss/grommet-core/_objects.tab.scss [hunk 1]
```diff
@@ -4,6 +4,24 @@
 
   padding: halve($inuit-base-spacing-unit);
 
+  .tabs--justify-start &:first-of-type,
+  .tabs--justify-end &:first-of-type {
+      padding-left: 0;
+
+      @include media-query(portable) {
```
Comment (ericsoderberghp) [inline src/scss/grommet-core/_objects.tab.scss]: Our breaks should always be "palm".

File: src/scss/grommet-core/_objects.tabs.scss [hunk 2]
```diff
@@ -5,16 +5,33 @@
...
 
   &__content {
-    padding: $inuit-base-spacing-unit;
-
     @include media-query(palm) {
       padding: $inuit-base-spacing-unit 0;
     }
   }
+
+  &--justify-center {
```
Comment (ericsoderberghp) [inline src/scss/grommet-core/_objects.tabs.scss]: We typically use SASS a bit more via:

&--justify {
  &-center {
    ...
  }
  &-start {
    ...
  }
...

File: src/scss/grommet-core/_objects.tabs.scss [hunk 3]
```diff
@@ -5,16 +5,33 @@
...
+    justify-content: center;
+  }
+
+  &--justify-start {
+    justify-content: flex-start;
+
+    @include media-query(portable) {
```
Comment (ericsoderberghp) [inline src/scss/grommet-core/_objects.tabs.scss]: Do we even need the media queries?

File: src/js/components/Tabs.js [hunk 4]
```diff
@@ -14,16 +14,34 @@ export default class Tabs extends Component {
     this._activateTab = this._activateTab.bind(this);
 
     this.state = {
-      activeIndex: props.initialIndex
+      activeIndex: props.initialIndex,
+      justify: props.justify
     };
   }
 
+  _addPropertyClass (classes, prefix, property, classProperty) {
```
Comment (ericsoderberghp) [inline src/js/components/Tabs.js]: Why do we need this function, it seems a bit overkill for just adding `${CLASS_ROOT}--justify-${this.props.justify}`.

File: src/js/components/Tabs.js [hunk 5]
```diff
@@ -52,6 +70,17 @@ export default class Tabs extends Component {
       activeTitle: activeTitle
     });
 
+    var tabsContentPad;
```
Comment (ericsoderberghp) [inline src/js/components/Tabs.js]: I think any padding should be controlled by the caller. I've discussed with Allan and he's planning to merge the div and Box and have no padding. Which would make Brand Central easier. :)

File: src/scss/grommet-core/_objects.tabs.scss (hunk 3 above)
Comment (jwijay) [inline src/scss/grommet-core/_objects.tabs.scss]: Still left-justified for small screens:
screen shot 2016-02-10 at 9 17 52 am
Center-justified for small screens:
screen shot 2016-02-10 at 9 18 12 am

I think the latter looks better if the screen is too small to support all tabs on a single row. What do you think?

Comment (ericsoderberghp) [inline src/scss/grommet-core/_objects.tabs.scss]: I don't think either of them look great. Instead, I'm inclined to convert them to direction="column" and just stack them, and add a responsive={false} to Tabs to turn off this behavior. That allows callers that have a few tabs with small names to still work horizontally.

Comment (jwijay) [inline src/scss/grommet-core/_objects.tabs.scss]: ok! I'll do that.

**Pull request merged**